test/tests/contacts/ContactUtilsTest.ts

```diff
@@ -177,4 +187,55 @@ o.spec("ContactUtilsTest", function () {
 		o(formatBirthdayNumeric(leapYearBirthday)).equals("29/02/2016")
 		o(formatBirthdayNumeric(chromeBugBirthday)).equals("15/08/1911")
 	})
+
+	o("extractStructuredEmailAddress", function () {
+		let contact = createTestEntity(ContactTypeRef)
```

Comment:
```suggestion
		const contact = createTestEntity(ContactTypeRef)
```
The reference does not seem to change here, so this can be a constant.